desc/compute/_geometry.py

```diff
@@ -194,7 +194,7 @@ def _R0_over_a(params, transforms, profiles, data, **kwargs):
     transforms={"grid": []},
     profiles=[],
     coordinates="",
-    data=["sqrt(g)", "g_tt"],
+    data=["sqrt(g)", "g_tt", "R"],
```

Comment:
FYI this change is unnecessary because `"sqrt(g)"` already depends on `"R"`. But if we always want to list all explicit dependencies that is fine.